docs-kits/kits/DCM-Kit/page_dcm-guidelines-solution-adoption.md

```diff
@@ -0,0 +1,85 @@
+---
```

Comment:
Great addition! Could you rename the page to "Operation View" to stay consistent with the general Kit structure?

Reply:
Fixed in https://github.com/eclipse-tractusx/eclipse-tractusx.github.io/pull/836/commits/8f7cc23eaa3c817c9f508c30eb95336e1b154e12
and than renamed the files for consistency https://github.com/eclipse-tractusx/eclipse-tractusx.github.io/pull/836/commits/b5a69b6a68eabe282f8a2f37a579685d65f9c69a
and than fixed links because of the renaming https://github.com/eclipse-tractusx/eclipse-tractusx.github.io/pull/836/commits/1334379d75a3d3725aa8a9c3dbb75d5dd3d7a5cc